LingoClash/LingoClash/Model/Questions/QuestionTypes/QuestionWrapper.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  QuestionWrapper.swift
+//  testGame
+//
+//  Created by Sherwin Poh on 3/4/22.
+//
+
+struct QuestionWrapper: Codable {
+    let question: Question
```

Comment:
This is a wrapper to enable questions to be encodable and decodable. Unfortunately, cannot find an easier way out of this.